Makefile

```diff
@@ -24,6 +24,10 @@ verify-podnetworkconnectivitychecks:
 	$(MAKE) -C pkg/operator/connectivitycheckcontroller verify
 
 test-e2e-encryption: GO_TEST_PACKAGES :=./test/e2e-encryption/...
+test-e2e-encryption: GO_TEST_FLAGS += -v
+test-e2e-encryption: GO_TEST_FLAGS += -timeout 4h
+test-e2e-encryption: GO_TEST_FLAGS += -p 1
```

Comment:
is this required ?

Reply:
Not right now since there is only one test in the package, but it can be a great protection in the future if we add another test as it would prevent running different encryption changes in parallel against the same cluster.

Based on that, do you think we should keep it?